src/Libraries/RevitNodes/Elements/InternalUtilities/ElementUtils.cs

```diff
@@ -48,5 +51,65 @@ public static void UpdateLevelName(ref string name)
                 name += "(1)";
             }
         }
+
+
+        /// <summary>
+        /// Get a revit parameters value
+        /// </summary>
+        /// <param name="param">Revit parameter</param>
+        /// <returns></returns>
+        public static object GetParameterValue(Autodesk.Revit.DB.Parameter param)
```

Comment:
isVisible won't stop the input parameter from being imported into the VM I don't think, so the revit type will still leak into the library search.